Helper/chords_usb.py

```
@@ -0,0 +1,156 @@
+import serial
+import time
+import numpy as np
+import serial.tools.list_ports
+import sys
```

Ruff (0.11.9)

`sys` imported but unused

Remove unused import: `sys`

(F401)